### DISCUSSION TITLE
Adding Metadata from Plugin results in error

### DISCUSSION THREAD

**Discussion by WorldTeacher** [Difegue/LANraragi#431] [Q&A] [answered]
**LRR Version and OS**
version: 0.7.7
OS: Windows 10

**Bug Details**
I'm trying to add metadata from either nhentai.net or e(x)hentai and It results in errors.
When adding an URL, I get this error:
nhentai: Error: Can't connect: Try again at /home/koyomi/lanraragi/script/../lib/LANraragi/Plugin/Metadata/nHentai.pm line 126. 
e(x)hentai: Error: can't connect, try again at /home/koyomi/lanraragi/script/../lib/LANraragi/Plugin/Metadata/EHentai.pm line 241. 
without URL:
nHentai: Error: Can't connect: Try again at /home/koyomi/lanraragi/script/../lib/LANraragi/Plugin/Metadata/nHentai.pm line 98
e(x)hentai: Error: Can't connect: Try again at /home/koyomi/lanraragi/script/../lib/LANraragi/Plugin/Metadata/EHentai.pm line 193. 
It might just me being unable to figure the settings out..., but since I tried various comics and neither worked, I think it might be a bug.
**Matching Logs**
nothing useful in the logs, just EH using the cookie data to access EH

**Screenshots**
![LRR bug](https://user-images.githubusercontent.com/41587052/111544410-b8201b00-8774-11eb-8d44-d5b9c58fb965.png)

**Comment by Difegue** [comment, marked as answer]
Looks like the WSL distro is unable to reach the Internet.  
I had another user with the same problem here: https://github.com/Difegue/LANraragi/issues/358, you might have some luck doing what he did, e.g check that your VPN/DNS settings aren't blocking WSL.

**Comment by WorldTeacher** [comment]
Thanks, setting my VPN so that all traffic is going throigh, instead of splitting the connection fixed this issue.

After that I tried it again, and it works fine for manual detection with and without a link, but when I try to do the mass tag option, it stops after a second with an error message. I'll post an image of the error later today.
